core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -32,36 +40,52 @@ contract DesignatedVoting is MultiRole, Withdrawable {
         finder = FinderInterface(finderAddress);
     }
 
+    /****************************************
+     *   VOTING AND REWARD FUNCTIONALITY    *
+     ****************************************/
+
     /**
      * @notice Forwards a commit to Voting.
+     * @param identifier uniquely identifies the feed for this vote. EG BTC/USD price pair.
+     * @param time specifies the unix timestamp of the price being voted on.
+     * @param hash is the keccak256 hash of the price you want to vote for and a random integer salt value.
```

Comment:
nit: The way this comment renders on the doc site looks weird with the word "is" because of the colon.
```
hash: is the keccak256 hash of the price you want to vote for and a random integer salt value.
```
What do you think about dropping the first `is` in all comments and starting all `@param` docstrings directly with what the param is? E.g., 
`@param hash keccak256 hash of the price and salt`

Reply:
+1

Reply:
I agree with this. I have gone through all the comments and removed these redundant filler words